src/components/LandingPage/ExperienceBlock/index.js

```diff
@@ -51,9 +51,18 @@ const ExperienceBlock = ({ data, size, backable }) => {
 };
 
 ExperienceBlock.propTypes = {
-  backable: PropTypes.bool,
-  data: PropTypes.object.isRequired,
-  size: PropTypes.oneOf(['s', 'm', 'l']),
+  backable: PropTypes.bool.isRequired,
+  data: PropTypes.shape({
+    _id: PropTypes.string.isRequired,
```

Comment:
it looks like `_id` is mapping from `id` which comes from api.

maybe we can "correct" it to `id`

Reply:
Updated